go/teams/handler.go

```diff
@@ -42,6 +83,87 @@ func HandleRotateRequest(ctx context.Context, g *libkb.GlobalContext, teamID key
 	})
 }
 
+func sweepOpenTeamResetMembers(ctx context.Context, g *libkb.GlobalContext,
+	team *Team, resetUsers []keybase1.TeamCLKRResetUser) (needRepoll bool, err error) {
+	// When CLKR is invoked because of account reset and it's an open team,
+	// we go ahead and boot reset readers and writers out of the team. Key
+	// is also rotated in the process (in the same ChangeMembership link).
+	defer g.CTrace(ctx, "sweepOpenTeamResetMembers", func() error { return err })()
+
+	// Go through resetUsers and fetch non-cached latest EldestSeqnos.
+	resetUserSeqnos := make(map[keybase1.UID]keybase1.Seqno)
+	for _, u := range resetUsers {
+		if _, found := resetUserSeqnos[u.Uid]; found {
+			// User was in the list more than once.
+			continue
+		}
+
+		arg := libkb.NewLoadUserArg(g).
+			WithNetContext(ctx).
+			WithUID(u.Uid).
+			WithPublicKeyOptional().
+			WithForcePoll(true)
+		upak, _, err := g.GetUPAKLoader().LoadV2(arg)
+		if err == nil {
+			resetUserSeqnos[u.Uid] = upak.Current.EldestSeqno
+		} else {
+			g.Log.CDebugf(ctx, "Could not load uid:%s through UPAKLoader: %s", u.Uid)
+		}
+	}
+
+	var needTeamReload bool
+
+	err = RetryOnSigOldSeqnoError(ctx, g, func(ctx context.Context, _ int) error {
+		if needTeamReload {
+			team2, err := Load(ctx, g, keybase1.LoadTeamArg{
+				ID:          team.ID,
+				Public:      team.ID.IsPublic(),
+				ForceRepoll: true,
+			})
+			if err != nil {
+				return err
+			}
+			team = team2
+		}
+		needTeamReload = true
```

Comment:
This reloading business makes sense. But here's a suggestion for a simpler way. That second `_ int` arg to the closure up there is the attempt number. So this could just be:
```
if attempt > 0 {
    team, err = Load(ctx, g, keybase1.LoadTeamArg{
        ID: team.ID,
        Public: team.ID.IsPublic(),
        ForceRepoll: true,
    }
    if err != nil { return err }
}
```

Reply:
ah, right! Thank you